client/jetpack-connect/test/authorize.js

```diff
@@ -304,46 +354,107 @@ describe( 'JetpackAuthorize', () => {
 	} );
 
 	describe( 'getRedirectionTarget', () => {
-		test( 'should redirect to pressable if partnerSlug is "pressable"', () => {
-			const renderableComponent = <JetpackAuthorize { ...DEFAULT_PROPS } />;
-			const component = shallow( renderableComponent );
-			component.setProps( {
-				partnerSlug: 'pressable',
-			} );
-			const target = component.instance().getRedirectionTarget();
+		let originalWindowLocation;
+
+		beforeEach( () => {
+			originalWindowLocation = global.window.location;
+			delete global.window.location;
+			global.window.location = {
+				href: 'http://wwww.example.com',
+				origin: 'http://www.example.com',
+			};
+		} );
+
+		afterEach( () => {
+			global.window.location = originalWindowLocation;
+		} );
+
+		test( 'should redirect to pressable if partnerSlug is "pressable"', async () => {
+			renderWithRedux(
+				<JetpackAuthorize
+					{ ...DEFAULT_PROPS }
+					authQuery={ {
+						...DEFAULT_PROPS.authQuery,
+						alreadyAuthorized: true,
+					} }
+					partnerSlug="pressable"
+					isAlreadyOnSitesList={ true }
```

Comment:
Could be a good opportunity to clean those redundant `={ true }` prop values up:

```suggestion
					isAlreadyOnSitesList
```

Reply:
👍🏻